pkgs/development/libraries/libgda/6.x.nix

```
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, intltool
+, meson
+, ninja
+, itstool
+, libxml2
+, python3
+, gtk3
+, json-glib
+, isocodes
+, openssl
+, gnome3
+, gobject-introspection
+, vala
+, libgee
+, sqlite
+, gtk-doc
+, yelp-tools
+, mysqlSupport ? false
+, libmysqlclient ? null
+, postgresSupport ? false
+, postgresql ? null
```

```suggestion
, mysqlSupport ? false
, libmysqlclient
, postgresSupport ? false
, postgresql
```

that messes with overriding inputs.

---

What do you mean?